contentcuration/contentcuration/tests/viewsets/test_assessmentitem.py

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
I kind of liked having them parallel in the same file, because they are performing very similar operations in many cases, and it's good to keep them in step. I was hoping that the difference in overall test case name would be enough of a signal.

Reply:
Seems to be just a matter of preference so marking as resolved.